src/plugins/discover/public/application/components/sidebar/discover_field_details.tsx

```diff
@@ -41,62 +41,66 @@ export function DiscoverFieldDetails({
-            <EuiLink onClick={() => onAddFilter('_exists_', field.name, '+')}>
-              {details.exists}
-            </EuiLink>
-          ) : (
-            <span>{details.exists}</span>
-          )}{' '}
-          / {details.total}{' '}
-          <FormattedMessage
-            id="discover.fieldChooser.detailViews.recordsText"
-            defaultMessage="records"
-          />
-        </EuiText>
-      )}
-      {details.error && <EuiText size="xs">{details.error}</EuiText>}
-      {!details.error && (
-        <div style={{ marginTop: '4px' }}>
-          {details.buckets.map((bucket: Bucket, idx: number) => (
-            <DiscoverFieldBucket
-              key={`bucket${idx}`}
-              bucket={bucket}
-              field={field}
-              onAddFilter={onAddFilter}
-            />
-          ))}
-        </div>
-      )}
+    <>
+      <div className="dscFieldDetails">
+        {details.error && <EuiText size="xs">{details.error}</EuiText>}
+        {!details.error && (
+          <div style={{ marginTop: '4px' }}>
+            {details.buckets.map((bucket: Bucket, idx: number) => (
+              <DiscoverFieldBucket
+                key={`bucket${idx}`}
+                bucket={bucket}
+                field={field}
+                onAddFilter={onAddFilter}
+              />
+            ))}
+          </div>
+        )}
 
-      {details.visualizeUrl && (
-        <>
-          <EuiLink
-            onClick={() => {
-              getServices().core.application.navigateToApp(details.visualizeUrl.app, {
-                path: details.visualizeUrl.path,
-              });
-            }}
-            className="kuiButton kuiButton--secondary kuiButton--small kuiVerticalRhythmSmall"
-            data-test-subj={`fieldVisualize-${field.name}`}
-          >
-            <FormattedMessage
-              id="discover.fieldChooser.detailViews.visualizeLinkText"
-              defaultMessage="Visualize"
-            />
+        {details.visualizeUrl && (
+          <>
+            <EuiLink
+              onClick={() => {
+                getServices().core.application.navigateToApp(details.visualizeUrl.app, {
+                  path: details.visualizeUrl.path,
+                });
+              }}
+              className="kuiButton kuiButton--secondary kuiButton--small kuiVerticalRhythmSmall"
+              data-test-subj={`fieldVisualize-${field.name}`}
+            >
+              <FormattedMessage
+                id="discover.fieldChooser.detailViews.visualizeLinkText"
+                defaultMessage="Visualize"
+              />
+            </EuiLink>
             {warnings.length > 0 && (
               <EuiIconTip type="alert" color="warning" content={warnings.join(' ')} />
             )}
-          </EuiLink>
-        </>
-      )}
-    </div>
+          </>
+        )}
+      </div>
+      <EuiPopoverFooter>
+        {!details.error && (
```

Comment:
I'd suggest wrapping the footer in this if statement as well or else you get an empty footer. 
<img width="360" alt="Screen Shot 2020-08-04 at 15 23 53 PM" src="https://user-images.githubusercontent.com/549577/89335817-85526800-d666-11ea-9ee9-37a57f1fd022.png">